src/Process.js

```diff
@@ -58,7 +58,7 @@ Process.prototype.ready = T.func([T.Number], T.Object, "process.ready").of(funct
 
     this.instance.onReady(deferred.resolve);
     this.instance.run();
-    setTimeout(deferred.reject, timeout, new Error("Timeout exceeded."));
+    setTimeout(() => this.instance.timeout(deferred), timeout);
```

Comment:
ist noch zu ändern von mir.

steht dort

`setTimeout(this.instance.timeout, timeout, deferred);`

Kam beim make der Fehler dass this.lastOutput() keine Funktion ist.